pyiron_lammps/compatibility/file.py

```
@@ -176,3 +190,25 @@ def lammps_file_initialization(structure, dimension=3, units="metal"):
         "read_data lammps.data",
     ]
     return init_commands
+
+
+def _modify_input_dict(
+    input_control_file: Optional[dict] = None,
+    lmp_str_lst: list[str] = [],
```

Ruff (0.14.6)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)